R/pkg/R/sparkR.R

```diff
@@ -381,6 +381,10 @@ sparkR.session <- function(
     deployMode <- sparkConfigMap[["spark.submit.deployMode"]]
   }
 
+  if (!exists("spark.sql.default.derby.dir", envir = sparkConfigMap)) {
```

Comment:
it might not be, but if you refer to the JIRA, our goal is only to change the derby output path since it is the default metastore